custom/icds_reports/utils/__init__.py

```diff
@@ -435,9 +435,11 @@ def get_counseling(value):
 
 def get_tt_dates(value):
     tt_dates = []
-    if value['tt_1']:
+    # ignore 1970-01-01 as that is default date for ledger dates
```

Comment:
seems like an unfortunate default. is that just because it's a timestamp set to 0?

Reply:
It comes from https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/indicators/__init__.py#L154-L164

I am not totally sure why that was done, but it's really in the DB as that date, not a zero value.